frontend/src/scenes/dashboard/dashboardLogic.js

```diff
@@ -20,7 +20,7 @@ export const dashboardLogic = kea({
 
```

Comment:
Maybe worth converting to ts as you're editing this?

Reply:
Will leave this for a separate PR, there are too many changes required (60+)